device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/sai_2700_48x50g_8x100g.xml

```diff
@@ -1,270 +1,286 @@
-<?xml version="1.0"?>

```

Comment:
Any reason why the entire file is shown as diff?

Reply:
Good question. I don't know.